contracts/controller/src/prices.rs

```diff
@@ -22,17 +23,27 @@ impl Contract {
         // Update & insert operation
         self.prices.insert(&price.asset_id, &price);
     }
+
+    pub fn get_markets(&self) -> HashMap<AccountId, AccountId> {
```

Comment:
@dmytro-shash Isn't `UnorderedMap` has not representation? Do we really need to convert it to hashmap?